packages/core/src/bags/redux/actions/reset.js

```diff
@@ -28,6 +28,39 @@ const resetEntities = () => dispatch => {
   });
 };
 
+/**
+ * Reset bag operations entities to its initial value.
+ *
+ * @private
+ * @function
+ * @memberof module:bags/actions
+ *
+ * @example
+ * // Store before executing action
+ * const store = {
+ *     entities: {
+ *         bag: { 123: {...} },
+ *         bagItems: { 1: {...} },
+ *         bagOperations: { 1: {...} }
+ *     }
+ * }
+ *
+ * // Result of reset bag operations entities:
+ * const store = {
+ *     entities: {
+ *         bag: { 123: {...} },
+ *         bagItems: { 1: {...} }
+ *     }
+ * }
+ *
+ * @returns {Function} Dispatch reset bag operations entities action.
+ */
+export const resetBagOperationsEntities = () => dispatch => {
```

Comment:
Just wondering if we need a thunk in here, or can we do a simple action creator?

Reply:
Do you see a use-case for a thunk here? I think the action creator is enough for a simple dispatch but let me know.

Reply:
Resolving. Using a thunk for consistency.